packages/codebytes/src/languageSelection.tsx

```diff
@@ -0,0 +1,45 @@
+import { Select } from '@codecademy/gamut';
+import { Background } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+import type { languageOption } from './consts';
+import { languageOptions } from './consts';
+
+const StyledSelect = styled(Select)`
```

Comment:
@dreamwasp has done more Select stuff than me, but I want to say we don't want to continue adding styles in this old form, and that this should use a dark mode Select?

Reply:
yupp, that sounds right to me!

Reply:
sounds good! will update this!